pallets/gear/src/lib.rs

```diff
@@ -422,14 +422,14 @@ pub mod pallet {
 
         /// Initialization
         fn on_initialize(bn: BlockNumberFor<T>) -> Weight {
-            log::debug!(target: "runtime::gear", "⚙️ Initialization of block #{:?}", bn);
+            log::debug!(target: "runtime::gear", "⚙️  Initialization of block #{:?}", bn);
```

Comment:
Why do we need to write two whitespaces to show one whitespace? Looks weird.

Reply:
Actually in my terminal without second space it looks like this:
<img width="595" alt="image" src="https://user-images.githubusercontent.com/11819936/193450260-c6fc63e3-1bbf-49ef-b6ab-bdd63b0b2d09.png">

With second space:
<img width="595" alt="image" src="https://user-images.githubusercontent.com/11819936/193450306-8700e175-4b33-43a8-adae-61c354e0088a.png">

Reply:
Agree, the gear (irony!) emoji seems to have some problem with kerning in the terminal.

The workaround in Substrate is the same: https://github.com/paritytech/substrate/blob/master/client/informant/src/display.rs#L118